pages/agent/cli_meta_data.md.erb

```diff
@@ -8,7 +8,7 @@ See the [Using Build Metadata](/docs/builds/build-meta-data) guide for a step-by
 
 ## Setting Data
 
-Use this command in your build scripts to save simple string data to the build for retrieval in future steps.
```

Comment:
I know that in this instance 'simple string data' is referring to the type of data that you want to receive - is it actually different to referring to 'string data'? Are we specifically requesting that they use a subset of string's capabilities?

Reply:
Nope, you are correct, it's just a string.